pkg/query/api/v1.go

```diff
@@ -192,7 +194,7 @@ func NewAPI(
 }
 
 // Register the API's endpoints in the given router.
-func (api *API) Register(r *route.Router, tracer opentracing.Tracer, logger log.Logger, ins extpromhttp.InstrumentationMiddleware) {
+func (api *API) Register(r *route.Router, tracer opentracing.Tracer, logger log.Logger, ins extpromhttp.InstrumentationMiddleware, logMiddleware *logging.ServerMiddleware) {
```

Comment:
maybe interface would be better so don't need to tie those packages together? :thinking:

Reply:
Yeah, but for the scope of this PR, I would implement this in another PR